icu4j/main/core/src/main/java/com/ibm/icu/text/UnicodeSet.java

```diff
@@ -278,14 +283,25 @@
  *     </tr>
  *   </table>
  * </blockquote>
- * <p>To iterate over contents of UnicodeSet, the following are available:
- * <ul><li>{@link #ranges()} to iterate through the ranges</li>
- * <li>{@link #strings()} to iterate through the strings</li>
- * <li>{@link #iterator()} to iterate through the entire contents in a single loop.
- * That method is, however, not particularly efficient, since it "boxes" each code point into a String.
+ *
+ * <p>To iterate over contents of {@code UnicodeSet}, the following are available:
+ * <ul>
+ *   <li>to iterate through the ranges: {@link #ranges()}, {@link UnicodeSetIterator#nextRange()},
+ *       {@link #rangeStream()}</li>
+ *   <li>to iterate through the strings: {@link #strings()}, {@link UnicodeSetIterator#next()},
+ *       {@link #stringStream()}</li>
+ *   <li>to iterate through the code points: {@link UnicodeSetIterator#next()},
+ *       {@link #codePoints()}, {@link #codePointStream()}</li>
+ *   <li>to iterate through the entire contents in a single loop: {@link #iterator()},
+ *       {@link UnicodeSetIterator#next()}, {@link #stream()}.<br>
+ *       All of these method are, however, not particularly efficient,
+ *       since they convert each individual code point to a {@code String}.
  * </ul>
- * All of the above can be used in <b>for</b> loops.
- * The {@link com.ibm.icu.text.UnicodeSetIterator UnicodeSetIterator} can also be used, but not in <b>for</b> loops.
+ *
+ * <p>The iterators and streams methods work as expected in current idiomatic Java usage.<br>
+ * The {@link UnicodeSetIterator} cannot be in <b>for</b> loops, and it is not very Java-ideomatic, because it is old.
+ * But it might be faster in certain use cases. We recommend that you measure in performance sensitive code.<br>
```

Comment:
```suggestion
 * <p>The iterators and streams methods work as expected in idiomatic Java usage.<br>
 * The {@link UnicodeSetIterator} cannot be used in <b>for</b> loops, and it is not very Java-idiomatic, because it is old.
 * But it might be faster in certain use cases. We recommend that you measure in performance sensitive code.<br>
```